go.mod

```diff
@@ -1,14 +1,15 @@
 module github.com/livekit/server-sdk-go
 
-go 1.17
+go 1.18
```

Comment:
Is this required? doing so would remove 1.17 compatibility, which we want to avoid for the SDK

Reply:
Should not be needed. Will check.